easybuild/tools/py2vs3/__init__.py

```diff
@@ -22,40 +22,13 @@
 # You should have received a copy of the GNU General Public License
 # along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
 #
-import sys
 
-# all functionality provided by the py2 and py3 modules is made available via the easybuild.tools.py2vs3 namespace
-if sys.version_info[0] >= 3:
-    from easybuild.tools.py2vs3.py3 import *  # noqa
-else:
-    from easybuild.tools.py2vs3.py2 import *  # noqa
+# all functionality provided by the py3 modules is made available via the easybuild.tools.py2vs3 namespace
+from easybuild.tools.py2vs3.py3 import *  # noqa
```

Comment:
Rather than doing this, we should remove the entire `py2vs3` package, and replace all imports accordingly?

That also implies removing things like our custom `string_type`, etc.

Reply:
While we've deprecated using EB on Python 2, we've not deprecated the use of `py2vs3`. Are we fine with making this change? Or should we deprecate `py2vs3` in EB5 and remove it in a later version.

Either way I'll switch over the imports.

Reply:
I guess we could debate over whether or not we'll keep `py2vs3` to soften the blow w.r.t. dropping support for Python 2...
Deprecating imports from `py2vs3` rather than removing it entirely is perhaps a better approach, indeed.